src/Form/FieldSelect.js

```diff
@@ -0,0 +1,198 @@
+import classNames from 'classnames/dedupe';
+import React from 'react';
+
+import Dropdown from '../Dropdown/Dropdown';
+
+class FieldSelect extends React.Component {
+
+  handleChange(selectedValue) {
+    this.props.handleEvent('change',
+      this.props.name, selectedValue);
+  }
+
+  hasError() {
+    let {props} = this;
+    let validationError = props.validationError;
+    return !!(validationError && validationError[props.name]);
+  }
+
+  getDropDown(dropdownItems) {
+    let {
+      defaultValue,
+      dropdownMenuButtonClassName,
+      dropdownMenuClassName,
+      dropdownMenuListClassName,
+      dropdownMenuWrapperClassName,
+      persistentID
+    } = this.props;
+
+    if (defaultValue == null) {
+      defaultValue = dropdownItems[0];
+    }
+
+    if (typeof defaultValue === 'string'){
+      defaultValue = {
+        html: defaultValue,
+        id: defaultValue.toLowerCase()
+      };
+    }
+
+    let dropdownMenuButtonClassSet = classNames('button dropdown-toggle',
```

Comment:
I do like the second approach, thanks for the suggestion will adjust the code to it.